tests/test_vanillaandbean.py

```diff
@@ -0,0 +1,53 @@
+from tests import ScraperTest
+
+from recipe_scrapers.vanillaandbean import VanillaAndBean
+
+
+class TestVanillaAndBeanScraper(ScraperTest):
+
+    scraper_class = VanillaAndBean
+
+    def test_host(self):
+        self.assertEqual("vanillaandbean.com", self.harvester_class.host())
+
+    def test_canonical_url(self):
+        self.assertEqual(
+            "https://vanillaandbean.com/maple-oat-sourdough-sandwich-bread/",
+            self.harvester_class.canonical_url(),
+        )
+
+    def test_title(self):
+        self.assertEqual(
+            self.harvester_class.title(), "Maple Oat Sourdough Sandwich Bread"
+        )
+
+    def test_yields(self):
+        self.assertEqual("1 serving(s)", self.harvester_class.yields())
+
+    def test_image(self):
+        self.assertEqual(
+            "https://vanillaandbean.com/wp-content/uploads/2020/06/aaasourdough-20.jpg",
+            self.harvester_class.image(),
+        )
+
+    def test_ingredients(self):
+        self.assertCountEqual(
```

Comment:
Hrm.  It looks like a number of the previously-written tests in the codebase use this `assertCountEqual` pattern, but I'm not sure whether it's a reliable way to check that the scraper is working.  What do you think about switching this over to use `assertEqual` (or some other mechanism) instead?

Reply:
Makes sense to me -- I'll do that for future tests.